### ISSUE TITLE
Missing mandatory fields on assignTestCaseExecutionTask()

### ISSUE THREAD

**Issue by kengsenghor** [Nyaran/testlink-xmlrpc#6]
In the testlink.js api file, below function need improvement, due to missing important options (testplanid and buildid).

```js
    /**
     * Unassign a Test Case execution
     *
     * @param {object}  options Options
     * @param {int}     options.testplanid Test Plan id
     * @param {string}  options.testcaseexternalid Test Case external id
     * @param {string}  [options.buildid] Buid id. If not present, buildname must be present.
     * @param {string}  [options.buildname] Build name. If not present, buildid must be present.
     * @param {string}  [options.platformid] Platform id. If not present and Test Case has platforms, platformname must be present.
     * @param {string}  [options.platformname] Platform name. If not present and Test Case has platforms, platformid must be present.
     * @param {string}  options.user User name
     *
     * @returns {object}  result Result object.
     */
    assignTestCaseExecutionTask(options) {
        checkMandatoryFields(options, ["testcaseexternalid", "version", "testprojectid", "executiontype"]);

        return this._performRequest('assignTestCaseExecutionTask', options)
    }
```

Without those missing fields, the calling codes will failed.

**Comment by Nyaran** [comment]
Thanks. Fixed on new version (1.9.18-3)

**Comment by kengsenghor** [comment]
Many thanks. The feature working as expected.
